src/site/layouts/page.drupal.liquid

```diff
@@ -3,9 +3,14 @@
 {% include "src/site/includes/alerts.drupal.liquid" %}
 {% include "src/site/includes/preview-edit.drupal.liquid" %}
 {% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+{% assign lang = 'en' %}
+
+{% if entityUrl.path contains "-esp" %}
+  {% assign lang = 'es' %}
+{% endif %}
```

Comment:
I like this but I think we'll need a more precise condition or strategy for applying this to the template. Just thinking that URL might just have `-esp` somewhere in the URL but not necessarily be entirely in Spanish.

This page for example is for the Spanish version of 10-10EZ, and part of it is in Spanish, but I don't think we'd want the whole `#content` being marked as Spanish -> https://www.va.gov/find-forms/about-form-10-10ez-espanol/

I found that page by search for `-esp` in the [sitemap](https://www.va.gov/sitemap.xml), fysa

Reply:
very good point and thanks for the sitemap info... we need to check if the url ends with `-esp` because that was what we agreed would be the suffix for all spanish content: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/information-architecture/ia-reviews/cto-spanish-content.md#ia-structure-urls-and-breadcrumbs-

added a filter to do just that :)